app/views/idv/finance/new.html.slim

```diff
@@ -6,10 +6,21 @@ p.italic Please provide only one of the following:
 = simple_form_for(idv_finance_form, url: idv_finance_path,
     html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
   = f.error_notification
-  = f.collection_radio_buttons :finance_type, Idv::FinanceForm.finance_type_choices,
-    :first, :last, required: true, item_wrapper_tag: :div
-  = f.input :finance_account, label: false
+  .mb3.js-finance-choice-cntnr
+    = f.collection_radio_buttons(:finance_type, Idv::FinanceForm.finance_type_choices,
+      :first, :last, required: true, item_wrapper_tag: :div) do |r|
+      = r.label(class: 'radio')
+        = r.radio_button
+        span.indicator
+        = r.text
 
+  - label_txt = t('idv.form.finance_unselected')
+  - Idv::FinanceForm::FINANCE_TYPES.each do |finance_type|
+    - if idv_params[finance_type].present?
+      - label_txt = t("idv.form.#{finance_type}")
```

Comment:
love it, added, and thank you for the code snippets -- super helpful to see Michelangelo in action :)